modules/apps/content-dashboard/content-dashboard-journal-impl/src/main/java/com/liferay/content/dashboard/journal/internal/item/JournalArticleContentDashboardItem.java

```diff
@@ -460,6 +472,52 @@ public boolean isViewable(HttpServletRequest httpServletRequest) {
 		);
 	}
 
+	private List<ContentDashboardItemVersionAction>
```

Comment:
@cgoncas, this is the only think I dislike. I think we can extract the code to reuse it, but I'm unsure how....